src/pydap/net.py

```diff
@@ -3,6 +3,10 @@
 from contextlib import closing
 import requests
 import ssl
+try:
+    from urllib import quote  # Python 2.x
```

Comment:
@jgallagher59701 and I had a quick chat, and I think we can remove this optional import for Python 2 support. The PyDAP CI/CD is only testing against Python 3 (3.7 or newer), and Python 2 sunset date was the 1st of January 2020.